akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/sse/LineParserSpec.scala

```diff
@@ -15,18 +15,21 @@ final class LineParserSpec extends AsyncWordSpec with Matchers with BaseUnmarsha
 
   "A LineParser" should {
 
-    "parse lines terminated with either CR, LF or CRLF" in {
+    "parse lines terminated with either LF or CRLF" in {
       Source
         .single(ByteString("line1\nline2\rline3\r\nline4\nline5\rline6\r\n\n"))
         .via(new LineParser(1048576))
         .runWith(Sink.seq)
         .map(_ shouldBe Vector("line1", "line2", "line3", "line4", "line5", "line6", ""))
+    }
 
+    "ignore lines terminated with CR" in {
```

Comment:
Hmm, is this something we're willing to accept?

Reply:
Looks like a bug to me.